index.d.ts

```diff
@@ -342,16 +342,16 @@ declare module 'fluture' {
 
   }
 
-  export var Future: Fluture
+  export var Future: FutureTypeRep
   export default Future
```

Comment:
It's all about this line here. This used to mean "export the `Future` *interface* as `default`", and now it means "export the `Future` *value* as `default`".